packages/forms-web-app/src/views/components/ui/results-per-page.njk

```diff
@@ -0,0 +1,12 @@
+{% macro uiResultsPerPage(resultsPerPage) %}
+	<p >
+		<span class="govuk-!-font-weight-bold govuk-!-margin-right-3">Results per page</span>
```

Comment:
In the past accessibility testing has picked us up on applying bold to tags suggesting to use `<strong>...</strong>` instead.